src/numbersFromStream/TestStream.java

```diff
@@ -0,0 +1,33 @@
+package numbersFromStream;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+public class TestStream {
+    public static void main(String[] args) {
+
+        Stream<Integer> numbers = Stream.iterate(8, number -> number + 1);
```

Comment:
Jak tworzysz `stream` to nie musisz go przypisywać do zmiennej - mogłaś wykonywać od razu na nim metody które wypisałaś sobie od linii #13 w dół.

Reply:
poprawione